app/i18n/de/conf.php

```diff
@@ -49,11 +49,11 @@
 			'thin' => 'Klein',
 		),
 		'thumbnail' => array(
-			'label' => 'Thumbnail', // TODO - Translation
-			'none' => 'None', // TODO - Translation
-			'portrait' => 'Portrait', // TODO - Translation
-			'square' => 'Square', // TODO - Translation
-			'landscape' => 'Landscape', // TODO - Translation
+			'label' => 'Vorschaubild',
+			'none' => 'Keins',
```

Comment:
keins or keine?

Reply:
I would say that "keins" is correkt. 
"kein" would be also correct. 
"keine" is not correct
![grafik](https://user-images.githubusercontent.com/1645099/132982006-167fda94-92f5-4205-8ddf-8b4bd2b2161c.png)